tests/infra/logging_app.py

```diff
@@ -2,61 +2,173 @@
 # Licensed under the Apache 2.0 License.
 
 import infra.checker
+import suite.test_requirements as reqs
+import time
 
 from loguru import logger as LOG
 
 
+@reqs.description("Running transactions against logging app")
+@reqs.supports_methods("LOG_record", "LOG_record_pub", "LOG_get", "LOG_get_pub")
+def test_run_txs(
+    network,
+    args,
+    logging_app,
```

Comment:
I believe that this won't work when plugged in the test suite. All arguments except `network` and `args` should be defaulted. We've come across this problem before and the solution is that you can attach a `txs` object to the network (see example in `recovery.py`).

Reply:
thanks, done